style.css

```diff
@@ -0,0 +1,120 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.navigation-bar {
+    display: flex;
+    width: 100%;
+    background-color: #A5B4FC;
+    justify-content: space-between;
+    padding: 0.9rem;
+}
```

Comment:
Mostly you will use same navbar so try creating separate style sheet so that you can import it in every page.